src/workerd/jsg/setup.h

```diff
@@ -656,9 +698,37 @@ class Isolate: public IsolateBase {
     });
   }
 
+ protected:
+  inline TypeWrapper* getWrapperByContext(jsg::Lock& js) {
+    if (KJ_LIKELY(!hasExtraWrappers)) {
+      return wrappers[0].get();
+    } else {
+      return getWrapperByContext(js.v8Context());
+    }
+  }
+  inline TypeWrapper* getWrapperByContext(v8::Local<v8::Context>&& context) {
```

Comment:
You wouldn't need this overload if the overload below took a plain value instead of a reference. `v8::Local` just wraps a pointer anyway, so passing it by value is not expensive.